### PR TITLE
Test case 01

### PR DESCRIPTION
**Pull request by sattdn210106**

### CODE REVIEW

File: src/main/java/common/Common.java [hunk 1]
```diff
@@ -1,22 +1,10 @@
...
 
 public class Common {
-    public static JSONArray getJsonData(String pathFileJson) throws IOException, ParseException {
-        JSONParser jsonParser = new JSONParser();
-        FileReader reader = new FileReader(pathFileJson);
-        Object obj = jsonParser.parse(reader);
-        return (JSONArray) obj;
-    }
 
     public static String plusDayFromNow(long daysToAdd, String formatDay) {
```
Comment (thanhlelgg) [inline src/main/java/common/Common.java]: formatDay -> dateFormat

Comment (sattdn210106) [reply src/main/java/common/Common.java]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

File: src/main/java/page_objects/LoginPage.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import common.helpers.ElementHelper;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+    //Locators
+    private final By txtEmail = By.id("username");
+    private final By txtPassword = By.id("password");
+    private final By btnLogin = By.xpath("//input[@title='Login']");
+    private final By lblErrorMsg = By.xpath("//p[@class='message error LoginForm']");
```
Comment (tanmle) [inline src/main/java/page_objects/LoginPage.java]: can we use css selector with a single class here?

File: src/main/java/page_objects/BasePage.java [hunk 3]
```diff
@@ -1,14 +1,30 @@
...
 import common.helpers.BrowserHelper;
-import common.helpers.ElementHelper;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
 public class BasePage {
 
     //Locators
+    private final By tabLogin = By.xpath("//span[text()='Login']");
+    private final By lblWelcomeMessage = By.xpath("//div[@class='account']");
```
Comment (tanmle) [inline src/main/java/page_objects/BasePage.java]: can we use By.className?

**Review by tanmle — APPROVED** [review]

File: src/main/java/page_objects/LoginPage.java (hunk 2 above)
Comment (sattdn210106) [reply src/main/java/page_objects/LoginPage.java]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

File: src/main/java/page_objects/BasePage.java (hunk 3 above)
Comment (sattdn210106) [reply src/main/java/page_objects/BasePage.java]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

**Pull request merged**